cmd/eval.go

```diff
@@ -223,11 +231,11 @@ func eval(args []string, params evalCommandParams) (err error) {
 
 	bs, err := readInputBytes(params)
 	if err != nil {
-		return err
+		return 2, err
```

Comment:
[nit] See above, I think on non-nil err, the other argument can be expected to _not be used_; and this thus wouldn't matter.